src/test/java/com/ravlinko/concordion/extension/mockserver/tag/StatusTagTest.java

```diff
@@ -0,0 +1,57 @@
+package com.ravlinko.concordion.extension.mockserver.tag;
+
+import org.concordion.api.CommandCall;
+import org.concordion.api.Element;
+import org.concordion.api.Evaluator;
+import org.concordion.api.ResultRecorder;
+import org.junit.Before;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.junit.Before'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)